openmetadata-ui/src/main/resources/ui/src/components/Chart/CustomBarChart.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { Col, Row, Typography } from 'antd';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Bar,
+  BarChart,
+  Legend,
+  ResponsiveContainer,
+  Tooltip,
+  XAxis,
+  YAxis,
+} from 'recharts';
+import { formatNumberWithComma } from '../../utils/CommonUtils';
+import ErrorPlaceHolder from '../common/error-with-placeholder/ErrorPlaceHolder';
+import { CustomBarChartProps } from './Chart.interface';
+
+const CustomBarChart = ({
+  chartCollection,
+  tickFormatter,
+  name,
+}: CustomBarChartProps) => {
+  const { data, information } = chartCollection;
+  const { t } = useTranslation();
+  const renderColorfulLegendText = (
+    value: string,
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    entry: any
+  ) => <span style={{ color: entry?.color }}>{value}</span>;
+
+  const tooltipFormatter = (value: string | number | (string | number)[]) => {
+    const numValue = value as number;
+
+    return (
+      <>
+        {tickFormatter
+          ? `${numValue.toFixed(2)}${tickFormatter}`
+          : formatNumberWithComma(numValue)}
+      </>
+    );
+  };
+
+  if (data.length === 0) {
+    return (
+      <Row
+        align="middle"
+        className="tw-h-full tw-w-full"
+        data-testid="no-data-placeholder"
+        justify="center">
+        <Col>
+          <ErrorPlaceHolder>
+            <Typography.Text>{t('label.no-data-available')}</Typography.Text>
+          </ErrorPlaceHolder>
+        </Col>
+      </Row>
+    );
+  }
+
+  return (
+    <ResponsiveContainer debounce={200} id={`${name}_graph`} minHeight={300}>
```

Comment:
Why we are adding debounce as a higher `200`?

Reply:
its 200 ms, so to creat graph we need to do some computation and it will take time so we need debounce